src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.html

```diff
@@ -1,14 +1,31 @@
-<cd-table [data]="hosts"
-          [columns]="columns"
-          columnMode="flex"
-          (fetchData)="getHosts($event)">
-  <ng-template #servicesTpl let-value="value">
-    <span *ngFor="let service of value; last as isLast">
-      <a [routerLink]="[service.cdLink]"
-         [queryParams]="cdParams"
-         *ngIf="service.canRead">{{ service.type }}.{{ service.id }}</a>
-      <span *ngIf="!service.canRead">{{ service.type }}.{{ service.id }}</span>
-      {{ !isLast ? ", " : "" }}
-    </span>
-  </ng-template>
-</cd-table>
+<tabset>
+  <tab i18n-heading
+       heading="Hosts List">
+       <cd-table [data]="hosts"
+                 [columns]="columns"
+                 columnMode="flex"
+                 (fetchData)="getHosts($event)"
+                 selectionType="single"
+                 (updateSelection)="updateSelection($event)">
+         <ng-template #servicesTpl let-value="value">
+           <span *ngFor="let service of value; last as isLast">
+             <a [routerLink]="[service.cdLink]"
+                [queryParams]="cdParams"
+                *ngIf="service.canRead">{{ service.type }}.{{ service.id }}</a>
+             <span *ngIf="!service.canRead">{{ service.type }}.{{ service.id }}</span>
+             {{ !isLast ? ", " : "" }}
+           </span>
+         </ng-template>
+         <cd-host-details cdTableDetail
+                 [selection]="selection">
+ </cd-host-details>
+       </cd-table>
+  </tab>
+
+  <tab heading="Overall Performance" i18n-heading>
```

Comment:
Please list every attribute in a separate line.
```
<tab heading="Overall Performance"
     i18n-heading>
```

Reply:
ack